<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseTrackbt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>285</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Choose a path</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pprint_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>640</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#000000;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scan">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>285</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Scan the path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputN">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> .QLineEdit {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Bebas Neue;
	font-size:12px;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter the song number.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onBT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>235</y>
      <width>110</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Turn on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="offBT_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>235</y>
      <width>110</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Turn off</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
